### ISSUE TITLE
Unable to open Google maps URL

### ISSUE THREAD

**Issue by mbnoimi** [osmandapp/Osmand#2456]
Whenever I open Google maps links I don't see open with OsmAnd choice as shown in the following screenshot (https://goo.gl/maps/G8LYVb9EYhy):
![screenshot_2016-04-14-16-25-44](https://cloud.githubusercontent.com/assets/284547/14529740/ad5fadde-025e-11e6-8fc1-77d314a64f3a.jpg)

- Android: 4.4.2
- OsmAnd: 2.2.3 re. 2016-03-24

**Comment by sonora** [comment]
See #2278. We do actual resolve many google map URL, but google has so many different ones in all differentt country domain that it is hard to support them all. many google.com do work, however.

**Comment by mbnoimi** [comment]
This is really awful. I'm unable to open any link shared by my friends!

I wonder why you don't support the used links by Google? What's the problem if Google has different types of links?

**Comment by andrew-davie** [comment]
I have to agree with Muhammad. It is shortsighted to reject such compatibility. I don’t understand why this request wasn’t given more attention. So what, there are a few variants… either you support all google links, or you’re basically not compatible at all.  I may very well have a go at fixing this myself; it’s broken and unsatisfactory!

**Comment by mhyqwvcu** [comment]
@andrew-davie see https://github.com/osmandapp/Osmand/pull/1061

**Comment by sonora** [comment]
I think you misunderstand ... nobody says we do not want to support all google links. We are just not aware there is any good documentation and/or stability in how google behaves. The link I pointed you to in the other issue documents where in our code we support which google link types, and of course this is also where you can fix any new syntax google uses when you discover any.

**Comment by sonora** [comment]
Regular google links meanwhile supported, remaining issue is Google link-shortener URLs (goo.gl)
